src/commands/generate/utils.ts

```diff
@@ -105,3 +105,17 @@ export function addDeprecatedToDocs<T extends DeprecatableDefinitions>(typeDefin
     }
     return typeDefintion.docs != null ? [typeDefintion.docs] : undefined;
 }
+
+export function addIncubatingDocs(
+    endpointDefinition: IEndpointDefinition,
+    existingDocs: string[] | undefined,
+): string[] | undefined {
```

Comment:
everything looks reasonable and consistent here - just curious why the signature of docs is a string array??

Reply:
Yeah I've done this for consistency based on the deprecated function as well.